generated/nidaqmx/nidaqmx.proto

```diff
@@ -68,6 +68,7 @@ service NiDAQmx {
   rpc CreateAIPosEddyCurrProxProbeChan(CreateAIPosEddyCurrProxProbeChanRequest) returns (CreateAIPosEddyCurrProxProbeChanResponse);
   rpc CreateAIPosLVDTChan(CreateAIPosLVDTChanRequest) returns (CreateAIPosLVDTChanResponse);
   rpc CreateAIPosRVDTChan(CreateAIPosRVDTChanRequest) returns (CreateAIPosRVDTChanResponse);
+  rpc CreateAIPowerChan(CreateAIPowerChanRequest) returns (CreateAIPowerChanResponse);
```

Comment:
I can't comment at the top of the file but these .proto files have a comment for the version which right now is:
```proto
// This file is generated from NI-DAQMX API metadata version 20.0.0
```

Can you update the api_version in [config.py to 23.0.0](https://github.com/ni/grpc-device/blob/main/source/codegen/metadata/nidaqmx/config.py#L3) to in turn bump it here as well?

I know this isn't apparent so I added an [issue to scrapigen](https://github.com/ni/grpc-device-scrapigen/issues/144) to move these config.py files over to scrapigen as it makes more sense to bump the version over there and copy over like the other main metadata files. But this is for all of the metadata sourced from scrapigen, not just daqmx, so for now I'd say just bump here in grpc-device and hopefully down the line we'll fix it up to move config.py over there.

Reply:
Updated the api_version in config.py file